packages/vscode-extension/src/controls/webviewPanel.ts

```diff
@@ -69,7 +65,7 @@ export class WebviewPanel {
         // Enable javascript in the webview
         enableScripts: true,
         retainContextWhenHidden: true,
-        localResourceRoots: [vscode.Uri.file(path.join(this.extensionPath, "out"))],
+        localResourceRoots: [vscode.Uri.file(path.join(ext.context.extensionPath, "out"))],
```

Comment:
is any side effect on webpack. such as the path is different after webpack?

Reply:
Tested and works well